Cpp_test/lib/file-handler.cpp

```diff
@@ -66,5 +70,13 @@ size_t FileHandler::get_file_size()
     }
 
 	return myst.st_size;
+} 
+
 
-} 
\ No newline at end of file
+long FileHandler::get_read_bytes(){
+	if(!fs)
```

Comment:
Not sure if I understand this part. if (!fs) is true, that means filestream failed to open the file right? Then is there anything to return from gcount? This may be correct, I just don't understand this.

Reply:
Oh, stupid me. I changed if condition and then forgot to push it. It is not !fs, the right one is if(!this->fs.eof() && (this->fs.bad() || this->fs.fail())).